test/run_tests.sh

```diff
@@ -301,7 +308,7 @@ function run_mp_op_tests {
   run_test "$CDIR/test_mp_mesh_reduce.py"
   run_test "$CDIR/test_mp_sync_batch_norm.py"
   run_test "$CDIR/test_fsdp_auto_wrap.py"
-  run_torchrun "$CDIR/test_mp_early_exit.py"
+  run_torchrun_1 "$CDIR/test_mp_early_exit.py"
```

Comment:
suggests to rename it to run_torchrun_single_process

Reply:
Thanks, updated to `run_torchrun_single_process `